Playbooks/playbook-Vulnerability_Handling_-_Nexpose.yml

```diff
@@ -623,3 +623,6 @@ view: |-
   }
 inputs: []
 outputs: []
+releaseNotes: "Added 'No test' to playbook to allow for modified CveReputation script."
```

Comment:
if the changes in the file are internal (tests, bug fixes) use the "-" mark as release notes